token/confidential-transfer/proof-generation/src/supply.rs

```diff
@@ -0,0 +1,31 @@
+use {
```

Comment:
Terribly sorry about this 🙏 . I mentioned it in the other thread above, but let's move this to an `account_info` module so that the token-client can easily call `generate_proof_data(...)`.